readthedocs/projects/forms.py

```diff
@@ -204,8 +204,9 @@ class Meta:
             'analytics_disabled',
             'show_version_warning',
             'single_version',
-            'external_builds_enabled',
             'privacy_level',
```

Comment:
I'm tempted to move this one just below "default branch".

Reply:
Yea, let's do that -- it's more important than these other settings for sure.